diff --git a/src/qlever/qleverfile.py b/src/qlever/qleverfile.py
--- a/src/qlever/qleverfile.py
+++ b/src/qlever/qleverfile.py
@@ -359,7 +359,7 @@ class Qleverfile:
             "--system",
             type=str,
             choices=Containerize.supported_systems() + ["native"],
-            default="docker",
+            default="native",
             help=(
                 "Whether to run commands like `index` or `start` "
                 "natively or in a container, and if in a container, "
